packages/editor/src/components/post-text-editor/index.js

```diff
@@ -47,14 +57,32 @@ export default function PostTextEditor() {
 		setIsDirty( true );
 	};
 
+	/**
+	 * Ensures no unwanted HTML ends up in the list of parsed blocks.
+	 */
+	const cleanBlocks = useCallback( function ( blocks ) {
```

Comment:
This is for the text editor for the full post. Do we need something similar when editing in HTML mode an individual block?

Reply:
That is cleaned in the `paste-handler`, I wonder if we just use the same filtering function as there.

Reply:
Ah I remember looking there now, the schema there is pretty limited so I think we should do it this way.